libraries/AP_Common/Bitmask.h

```diff
@@ -65,7 +72,16 @@ class Bitmask {
         }
         return true;
     }
-    
+
+    // return number of bits set
+    uint16_t count() {
+        uint16_t sum = 0;
+        for (uint16_t i=0; i<numwords; i++) {
+            sum += bits[i];
```

Comment:
You're right.  I've changed this to use the __builtin_popcount feature.  I have serious concerns though that there will be a problem because Bitmask may allocate slightly more bits that the user has asked for (i.e. if you ask for 33 you get 64).  This means the ::count can return a number higher than the total number of bits the user requested and inside AP_SafeRTL::routine_cleanup this leads to an error.